src/App.jsx

```diff
@@ -1,33 +1,36 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
 
+let todoList = [
```

Comment:
Considering this array is not changing, you could alternatively use a constant (`const`) instead of `let` here.

More information can be found [here ](https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/)if you're interested.